cookbooks/aws-parallelcluster-install/recipes/intel_mpi.rb

```diff
@@ -43,7 +49,7 @@
   creates intelmpi_installation_path.to_s
 end
 
-append_if_no_line "append intel modules file dir to modules conf" do
+append_if_no_line "append intel modules file dir to modules conf #{node['cluster']['modulepath_config_file']}" do
```

Comment:
The fact that the path changes in environment-modules 4.x let me think that if there is an update of this package in other OSes we'll be broken for that os as well.

I'm thinking at least in our kitchen spec tests we can have a  test to verify environment-modules version in every OS, to be sure to fail if the package is updated.

Reply:
I did not noticed but also ubuntu uses environment-modules version 4.x.x but they have solved the issue of compatibility adding directly in the package a link that point  `/etc/environment-modules/modulespath` to `/usr/share/modules/init/.modulespath`

There is already a test that verify that the intel_mpi module added in the cookbook is available